packages/ui/src/CopyToClipboardButton/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import userEvent from '@testing-library/user-event';
+import { CopyToClipboardButton } from '.';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { PenumbraUIProvider } from '../PenumbraUIProvider';
+
+describe('<CopyToClipboardButton />', () => {
+  beforeAll(() => {
+    userEvent.setup();
+  });
+
+  it('copies the value of the `text` prop to the clipboard', async () => {
+    const { getByLabelText } = render(<CopyToClipboardButton text='copy that' />, {
+      wrapper: PenumbraUIProvider,
+    });
+
+    fireEvent.click(getByLabelText('Copy'));
+
+    const clipboardText = await navigator.clipboard.readText();
```

Comment:
This is made possible by the install of `@testing-library/user-event`, as I commented on above.